modules/libmodbus/3.1.10/patches/add_build_file.patch

```diff
@@ -0,0 +1,78 @@
+--- /dev/null
++++ a/BUILD.bazel
+@@ -0,0 +1,74 @@
++load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
++
++cc_library(
++    name = "modbus",
++    srcs = glob(["src/*.c"]) + [
++        ":generate_config_h",
++        ":generate_modbus_version_h",
++    ],
++    hdrs = glob(["src/*.h"]),
++    copts = ["-DHAVE_CONFIG_H"],
++    includes = ["src"],
++    include_prefix = "libmodbus",
++    visibility = ["//visibility:public"],
++)
++
++cc_binary(
++    name = "libmodbus.so.0",
++    srcs = [
++    ],
++    linkopts = [
++        "-Wl,-soname,libmodbus.so.0",
++        # Linker script to hide non-modbus symbols.
++        "-Wl,--version-script",
++        "$(location modbus-versionscript.lds)",
++    ],
++    linkshared = 1,
++    deps = [
++        ":modbus",
++        ":modbus-versionscript.lds",
++    ],
++    visibility = ["//visibility:public"],
```

Comment:
Do users have a reason to depend on this directly rather than through `modbus_shared`? If not, consider making the target private to allow for more flexibility in the future without the risk of breaking anyone.